### ISSUE TITLE
How to use mocha-parallel-test in grunt

### ISSUE THREAD

**Issue by panishvp** [mocha-parallel/mocha-parallel-tests#214]
Hi there,
I have a grunt file which isrunning mocha tests and creating junit report using reporter mocha-junit-reporter. Since the testsare taking too much of time i want to run these tests in parallel using mocha-parallel-tests. 

Right now i am using these versions:
"mocha": "~4.0.1"
"mocha-parallel-tests":"2.0.6-alpha.3"
 "mocha-junit-reporter": "^1.18.0",

**Comment by 1999** [comment]
Closing as a duplicate of #193
